build/params_2k.go

```diff
@@ -70,6 +70,9 @@ var UpgradeWatermelonHeight = abi.ChainEpoch(200)
 // This fix upgrade only ran on calibrationnet
 const UpgradeWatermelonFixHeight = -100
 
+// This fix upgrade only ran on calibrationnet
+const UpgradeWatermelonFix2Height = -101
```

Comment:
Let's rename this (and the above) fields to UpgradeCalibrationWatermelonFix2?

Reply:
Sorry about missing these !